Makefile.common_go

```diff
@@ -2,6 +2,10 @@
 # components/xyz-service/Makefile
 REPOROOT?=../..
 
+# Convert, e.g., "src/github.com/chef/automate/components/teams-service" to "components/teams-service"
+REPO_DIR := $(shell cd $(REPOROOT) && pwd)/
+COMPONENT_DIR := $(shell echo $${PWD\#"$(REPO_DIR)"})
```

Comment:
The tricky part here was using make variables as shell variables.
The first line gets the expanded path of the REPOROOT.
The second line then subtracts that prefix from the current directory using the shell variable syntax `${variable-name#prefix-to-strip}`

Reply:
I wonder if we could use `$(CURDIR)` at line 46 instead of doing this conversion.  This works for me locally though, so dealers choice.

Reply:
Thanks for the suggestion! I was not aware of `CURDIR`. That would work, yes, but it has the same (minor) drawback that using just `PWD` above would have; output is more verbose. One would see this:
```
/Users/msorens/code/go/src/github.com/chef/automate/components/teams-service [go]... CSpell: Files checked: 0, Issues found: 0 in 0 files
/Users/msorens/code/go/src/github.com/chef/automate/components/teams-service [rb]... CSpell: Files checked: 0, Issues found: 0 in 0 files
/Users/msorens/code/go/src/github.com/chef/automate/components/teams-service [md]... CSpell: Files checked: 0, Issues found: 0 in 0 files
/Users/msorens/code/go/src/github.com/chef/automate/components/teams-service [proto]... CSpell: Files checked: 0, Issues found: 0 in 0 files
/Users/msorens/code/go/src/github.com/chef/automate/components/teams-service [sh]... CSpell: Files checked: 0, Issues found: 0 in 0 files
```
instead of this:
```
components/teams-service [go]... CSpell: Files checked: 19, Issues found: 0 in 0 files
components/teams-service [rb]... CSpell: Files checked: 0, Issues found: 0 in 0 files
components/teams-service [md]... CSpell: Files checked: 3, Issues found: 0 in 0 files
components/teams-service [proto]... CSpell: Files checked: 0, Issues found: 0 in 0 files
components/teams-service [sh]... CSpell: Files checked: 1, Issues found: 0 in 0 files
```